src/components/composite/AssetOverviewCard/fixtures.ts

```
@@ -266,118 +67,496 @@ export const tokens: MinimalToken[] = [
-    lpToken: lpTokens.aEthWETH,
-    slug: 'aave-v3-weth',
-    featured: true,
-    forYou: false,
-    latest: latestData.aaveWeth,
-    balance: 1.8,
-    totalPriceUSD: 6210.24,
-    relatedPositions: [
+    name: 'Aave V3',
+    assetUsd: 7.04781961968e-9,
+    debtUsd: 1,
+    netUsd: -0.99999999295218038,
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 0.9999999929521803

(lint/correctness/noPrecisionLoss)